frontend/src/pages/ResetPassword.jsx

```diff
@@ -0,0 +1,61 @@
+import { useState } from "react";
+import Header from "../components/ui/Header";
+import { useLocation, useNavigate } from "react-router-dom";
+
+const ResetPasswordPage = () => {
+
+  const navigate = useNavigate()
+
+	const location = useLocation();
+	const { email } = location.state || {};
+
+  // temporary mock data
+  const mock_data = [
+    { email: "test@gmail.com", password: "password123" },
+    { email: "abc@icloud.com", password: "xyzpassword" },
+  ];
+
+	const [newPass, setNewPass] = useState(-1);
+
+  // temporary until backend is connected
+  function updatePassword() {
+    const user = mock_data.find((user) => user.email == email)
+    user.password = newPass;
+    console.log(`${email} has new password ${user.password}`)
+  }
+
+	function handleSubmit(e) {
+		e.preventDefault();
+    updatePassword();
+		navigate('/login');
+	}
+
+  return (
+		<div className="flex flex-col h-screen w-full min-w-[1024px] bg-white">
```

Comment:
Minor style issue: indentation is off a little at the beginning of the code for the actual page